pygfx/renderers/wgpu/meshshader.py

```diff
@@ -192,11 +206,33 @@ def code_vertex(self):
         fn get_sign_of_det_of_4x4(m: mat4x4<f32>) -> f32 {
             // We know/assume that the matrix is a homogeneous matrix,
             // so that only the 3x3 region is relevant for the determinant,
-            // which is faster to calculate that the det of the 4x4.
+            // which is faster to calculate than the det of the 4x4.
             let m3 = mat3x3<f32>(m[0].xyz, m[1].xyz, m[2].xyz);
             return sign(determinant(m3));
         }
 
+        fn dist_pt_line(x1: f32, y1: f32, x2: f32, y2: f32, x3: f32, y3: f32) -> f32 {
+         // Distance of pt (x3,y3) to line with coords(x1,y1) (x2,y2)
+         return abs((x2 - x1) * (y1 - y3) - (x1 - x3) * (y2 - y1)) / sqrt((x2-x1)*(x2-x1) + (y2-y1)*(y2-y1));
+        }
+
+        fn dist_pt_line_segment(x1: f32, y1: f32, x2: f32, y2: f32, x3: f32, y3: f32) -> f32 {
```

Comment:
Looks like this function is not used.

Reply:
yes i left it there to see if it makes a difference but it doesnt I will remove it.